src/samplers/powerspherical.jl

```diff
@@ -0,0 +1,27 @@
+# Sampler for Power Spherical
+
+struct PowerSphericalSampler{T <: Real} <: Sampleable{Multivariate,Continuous}
+    μ::Vector{T}
+    κ::T
+    dist_b::Beta
```

Comment:
Without a type annotation, the `_rand!` method is type-unstable.
```suggestion
    dist_b::Beta{T}
```